usr/lib/linuxmint/mintUpdate/mintUpdate.py

```diff
@@ -651,6 +651,33 @@ def check_policy(self):
                 break
         return (mint_layer_found, error_msg)
 
+    @staticmethod
+    def get_eol_status(early_warning_days=90):
```

Comment:
If the arg is not used, you're better off using a constant for it so it's clearer when looking at the code that the value doesn't depend on the call but is something that can be tuned.

Reply:
changed